src/components/Form.js

```diff
@@ -1,5 +1,7 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
+import _pick from "lodash/pick";
```

Comment:
Should we be calling `lodash.pick` and `lodash.get` (and adding `lodash.get` to `package.json`) instead?

Reply:
Looks like they are both being used, but I forgot to add lodash.get to the package json when I was moving the code over from the other branch. Will update it